lib/hydra/derivatives/processors/jpeg2k_image.rb

```diff
@@ -73,20 +73,18 @@ def layer_rates(layer_count, compression_numerator)
         rates.map(&:to_s).join(',')
       end
 
-      protected
```

Comment:
Thanks for the follow-up. I missed the context that these were class methods as well. I'm all for this change.